File: benchmark/results/java-invocation-script/run-004/b-baseline/output/com/metaengine/demo/notification/services/NotificationRepository.java
package com.metaengine.demo.notification.services;

import com.metaengine.demo.notification.aggregates.Notification;
import java.util.List;

/** NotificationRepository persistence service in the notification domain. */
public class NotificationRepository {
    /** Persist a new Notification from a partial input. */
    public Notification create(Notification input) {
        throw new UnsupportedOperationException("not implemented");
    }

    /** Find a Notification by id, or null if not found. */
    public Notification findById(String id) {
        throw new UnsupportedOperationException("not implemented");
    }

    /** List Notifications up to limit. */
    public List<Notification> list(int limit) {
        throw new UnsupportedOperationException("not implemented");
    }

    /** Delete a Notification by id. */
    public void delete(String id) {
        throw new UnsupportedOperationException("not implemented");
    }
}

File: benchmark/results/python-multilang/run-004/a-mcp/output/notification/services/notification_repository.py
from typing import Optional
from notification.aggregates.notification import Notification

"""NotificationRepository service."""
class NotificationRepository:
    def create(self, input: Notification) -> Notification:
        raise NotImplementedError('not implemented')
    def findById(self, id: str) -> Optional[Notification]:
        raise NotImplementedError('not implemented')
    def list(self, limit: float) -> list[Notification]:
        raise NotImplementedError('not implemented')
    def delete(self, id: str) -> None:
        raise NotImplementedError('not implemented')

